### DISCUSSION TITLE
Add editor for oci.conf file

### DISCUSSION THREAD

**Discussion by NucciTheBoss** [canonical/hpc-team#7] [General]
_oci.conf_ is used to configure container runtimes for Slurm. This functionality was added in 23.02. An editor should be added for this file since it enables Slurm cluster operators to set up containers with the scheduler. I'd also like to be able to easily configure a container runtime in the cluster.

Manual page: https://manpages.ubuntu.com/manpages/noble/en/man5/oci.conf.5.html

**Comment by jamesbeedy** [comment]
Great! Agreed!

**Comment by NucciTheBoss** [comment]
`slurmutils` now has an _oci.conf_ configuration file editor as of release [1.1.0](https://github.com/charmed-hpc/slurmutils/commit/aeeb786392ecc3bbf87369efb441a6b2a2c1b198). Closing the out discussion.
